app-admin/jenkins/autobuild/postinst

```diff
@@ -0,0 +1,3 @@
+systemd-sysusers jenkins.conf
+systemd-tmpfiles --create jenkins.conf
+[[ -d var/lib/jenkins ]] || install -dm 750 -o jenkins -g jenkins var/lib/jenkins
```

Comment:
This should be handled by tmpfiles.d.

Reply:
In reality, it didn't.
I have to manually run the two systemd commands to create the user/group and tmpfile folder.

Reply:
changed
now handled by tmpfiles.d